catanatron/game.py

```diff
@@ -295,6 +298,22 @@ def execute(self, action, initial_build_phase=False):
             player_to_act.resource_deck += cards_selected
             player_to_act.development_deck.draw(1, DevelopmentCard.YEAR_OF_PLENTY)
             self.resource_deck -= cards_selected
+        elif action.action_type == ActionType.PLAY_MONOPOLY:
+            player_to_act = action.player
+            card_type_to_steal = action.value
+            cards_stolen = ResourceDeck()
+            if not player_to_act.has_monopoly_card():
+                raise ValueError("Player doesn't have monopoly card")
+            for player in self.players:
+                if not player_to_act.color == player.color:
```

Comment:
thought this was easier than looping with indexes and checking against the `current_player_index` 🤷

Reply:
Looks good to me.